code/game/machinery/computer/computer.dm

```diff
@@ -11,6 +11,7 @@
 	integrity_failure = 100
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 40, "acid" = 20)
 	var/obj/item/circuitboard/circuit = null //if circuit==null, computer can't disassembly
+	var/frame_type = /obj/structure/computerframe
```

Comment:
Im quite iffy on the idea of adding an entire frame type var to each computer in the game just for the sake of a lavaland ruin.

Reply:
definitely true, but the var could be used in future if any other computers want to have bananium frames, of if new frame types were added. walls have the `girder_type` var which is almost never used as anything other than the default, but is still useful for future

Reply:
that being said, is there a better way to do this?